hazelcast/src/main/resources/hazelcast-config-3.8.xsd

```diff
@@ -2438,35 +2446,27 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="invalidate-on-change" type="xs:boolean" minOccurs="0" maxOccurs="1" default="true">
-                <xs:annotation>
-                    <xs:documentation>
-                        True to evict the cached entries if the entries are changed (updated or removed).
-                        Default value is true.
-                    </xs:documentation>
-                </xs:annotation>
-            </xs:element>
-            <xs:element name="in-memory-format" type="in-memory-format" minOccurs="0" maxOccurs="1" default="BINARY">
+            <xs:element name="max-size" type="xs:unsignedInt" minOccurs="0" maxOccurs="1" default="0">
                 <xs:annotation>
                     <xs:documentation>
-                        Data type used to store entries.
-                        Possible values:
-                        BINARY (default): keys and values are stored as binary data.
-                        OBJECT: values are stored in their object forms.
-                        NATIVE: keys and values are stored in native memory. Only available on Hazelcast Enterprise.
+                        Maximum size of the Near Cache. When max size is reached,
+                        cache is evicted based on the policy defined.
+                        Any integer between 0 and Integer.MAX_VALUE.
+                        0 means Integer.MAX_VALUE. Default is 0.
+
+                        Deprecated since 3.8, please use &lt;eviction/&gt;
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
             <xs:element name="cache-local-entries" type="xs:boolean" minOccurs="0" maxOccurs="1" default="false">
                 <xs:annotation>
                     <xs:documentation>
-                        True to cache local entries also.
+                        True to cache local entries, which belong to the member itself.
```

Comment:
This should be the only content change in the XSD files, everything else is just re-ordering.